1.0/test/imagestream.json

```diff
@@ -3,5 +3,20 @@
   "apiVersion": "v1",
   "metadata": {
     "name": "wildflyswarm-10-centos7"
-  }
+  },
```

Comment:
It seems that i doesn't affect the test and it's safe to remove build-config.json and imagestream.json